apps/src/templates/Notification.jsx

```diff
@@ -119,7 +119,8 @@ class Notification extends Component {
     dismissible: PropTypes.bool.isRequired,
     newWindow: PropTypes.bool,
     analyticId: PropTypes.string,
-    isRtl: PropTypes.bool.isRequired,
+    // isRtl not required so it can be used on Pegasus
+    isRtl: PropTypes.bool,
```

Comment:
YourSchool gets isRtl from Redux and it's on Pegasus.  Can we do something similar: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/code.org/pages/public/yourschool.js#L7

Reply:
well that was easy to fix. thanks for the link.